src/app/portlets/content-types/common/content-types-form/content-types-form.component.ts

```diff
@@ -79,6 +86,18 @@ export class ContentTypesFormComponent extends BaseComponent {
 
         this.messageService.messageMap$.subscribe(res => {
             this.actionButtonLabel = this.isEditMode ? this.i18nMessages['update'] : this.i18nMessages['save'];
+            this.formOptions = [
+                {
+                    command: this.toggleForm.bind(this),
+                    label: this.i18nMessages['edit']
+                },
+                {
+                    command: () => {
```

Comment:
The anonymous function is needed because they Emit event can be triggered when the component read the dialog configuration.